exercise/static/exercise/results_staff.js

```diff
@@ -0,0 +1,783 @@
+                        return exercise.id === exerciseID;
+                    })[0];
+
+                    points[module.name] = points[module.name] + exercise.points || exercise.points;
+                    totalPoints[module.name] = totalPoints[module.name] + exercise.points || exercise.points;
+                    if (moduleChecklist.indexOf(module) === -1) {
+                        
+                        moduleChecklist.push(module);
+                        if (module.submission_count > 0) {
+                            totalSubmissions[module.name] = totalSubmissions[module.name] + module.submission_count || module.submission_count;
+                            totalSubmitters[module.name] = totalSubmitters[module.name] + 1 || 1;
+                            if (points[module.name] === maxPoints[module.name]) {
+                                totalMaxSubmitters[module.name] = totalMaxSubmitters[module.name] + 1 || 1;
+                            }
+                        }
+                    }
+                });
+            }
+
+            // Calculate points for each exercise
+            if (showMethod === "all") {
+                $(_exerciseSelection).each(function() {
+                    let moduleID = $(this).data("moduleId");
+                    let exerciseID = $(this).data("exerciseId");
+                    let exercise = _points[sid].modules[moduleID - 1].exercises.filter(function(exercise) {
+                        return exercise.id === exerciseID;
+                    })[0];
+
+                    points[exercise.name] = exercise.points;
+                    totalPoints[exercise.name] = totalPoints[exercise.name] + exercise.points || exercise.points;
+                    if (exercise.submission_count > 0) {
+                        totalSubmissions[exercise.name] = totalSubmissions[exercise.name] + exercise.submission_count || exercise.submission_count;
+                        totalSubmitters[exercise.name] = totalSubmitters[exercise.name] + 1 || 1;
+                        if (exercise.points === exercise.max_points) {
+                            totalMaxSubmitters[exercise.name] = totalMaxSubmitters[exercise.name] + 1 || 1;
+                        }
+                    }
+                    
+                });
+            }
+
+            // Create the table row for a single student from the points above
+            htmlTablePoints += '<tr><td class="student-id stick-on-scroll">' + student.student_id + '</td>';
+            htmlTablePoints += '<td class="student-name stick-on-scroll">' + _users[sid].full_name + '</td>';
+
+            if (pointKeys.length > 0) {
+                let tagHtml = "";
+                let studentTags = student.tag_slugs;
+    
+                // Sort tags in order. Put internal and external tags in the beginning.
+                studentTags.sort(function(left, right){ 
```

Comment:
This should be handled, if you keep the order from the API.

Reply:
Yep, seems to work without the sorting.